<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeApp</class>
 <widget class="QMainWindow" name="RecipeApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RecipeApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>Return</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="homePage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>200</number>
        </property>
        <property name="topMargin">
         <number>100</number>
        </property>
        <property name="rightMargin">
         <number>200</number>
        </property>
        <property name="bottomMargin">
         <number>100</number>
        </property>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>Browse Recipes</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">pageSelectionButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createButton">
          <property name="text">
           <string>Create Recipe</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">pageSelectionButtons</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="favoritesButton">
          <property name="text">
           <string>Favorites</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">pageSelectionButtons</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="browsePage"/>
      <widget class="QWidget" name="recipePage"/>
      <widget class="QWidget" name="createPage"/>
      <widget class="QWidget" name="favoritesPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="pageSelectionButtons"/>
 </buttongroups>
</ui>
